EnglishTranslation-master/SKILL.tsv

```diff
@@ -9595,9 +9595,9 @@ SKILL_20170912_009594	Temporarily increases evasion when riding a Companion.	컴
 SKILL_20170912_009595	Evasion: +#{CaptionRatio}#%{nl}Duration: #{CaptionTime}# seconds	회피율 증가 #{CaptionRatio}#%{nl}지속시간 #{CaptionTime}#초	SKILL_20161005_006650	Evasion Rate: +#{CaptionRatio}#%{nl}Movement Speed: -#{CaptionRatio2}#	회피율 증가 #{CaptionRatio}#%{nl}이동속도 감소 #{CaptionRatio2}#	
 SKILL_20170912_009596	Set up a roost where the hawk can sit. The hawk will wait for the owner's order on the roost. The roost disappears when you move away from it.	매가 대기할 수 있는 횃대를 설치합니다. 횃대에 앉은 매는 주인의 지령을 상시 대기합니다. 횃대에서 멀어지면 횃대는 사라집니다.	SKILL_20150414_003815	Set up a roost where the hawk can sit. The hawk will wait for the owner's order on the roost. The roost disappears when you move away from it.	매가 대기할 수 있는 횃대를 설치합니다. 횃대에 앉은 매는 매는 주인의 지령을 상시 대기합니다. 횃대에서 멀어지면 횃대는 사라집니다.	
 SKILL_20170912_009597	Additional Damage: +#{CaptionRatio}#{nl}Number of Hits: #{CaptionRatio2}#{nl}Allies Affected: #{CaptionTime}#{nl}Duration: 300 seconds{nl}Consumes 1 Holy Powder	추가 대미지 +#{CaptionRatio}#{nl}공격 횟수 #{CaptionRatio2}#회{nl}아군 #{CaptionTime}#명에게 추가 부여{nl}지속시간 300초{nl}홀리 파우더 1개 소비	SKILL_20160718_006383	Additional Damage: +#{CaptionRatio}#{nl}Number of Hits: #{CaptionRatio2}#{nl}Allies Affected: #{CaptionTime}#{nl}Duration: 60 seconds{nl}Consumes 1 Holy Powder	추가 대미지 +#{CaptionRatio}#{nl}공격 횟수 #{CaptionRatio2}#회{nl}아군 #{CaptionTime}#명에게 추가 부여{nl}지속시간 60초{nl}홀리 파우더 1개 소비	
-SKILL_20170912_009598	Decreases Slash, Pierce, Bow, Gun and Magic attack damage for you and your party members.	자신과 파티원들의 베기, 찌르기, 활, 총, 마법 공격에 대한 피해를 감소 시킵니다.	SKILL_20170512_009442	Decreases Slash, Pierce, Bow and Gun attack damage for you and your party members.	자신과 파티원들의 베기, 찌르기, 활, 총 공격에 대한 피해를 감소 시킵니다.	
+SKILL_20170912_009598	Decreases Slash, Pierce, Bow, Gun and Magic Attack Damage for you and your party members.	자신과 파티원들의 베기, 찌르기, 활, 총, 마법 공격에 대한 피해를 감소 시킵니다.	SKILL_20170512_009442	Decreases Slash, Pierce, Bow and Gun attack damage for you and your party members.	자신과 파티원들의 베기, 찌르기, 활, 총 공격에 대한 피해를 감소 시킵니다.	
 SKILL_20170912_009599	HP Recovery: +#{CaptionRatio}#{nl} Aura Duration: 300 seconds	HP 회복력 #{CaptionRatio}# 증가{nl}아우라 지속시간 300초	SKILL_20150317_001538	HP Recovery: +#{CaptionRatio}#{nl} Aura Duration: 60 seconds	HP 회복력 #{CaptionRatio}# 증가{nl}아우라 지속시간 60초	
-SKILL_20170912_009600	Magic circle duration #{CaptionTime}# seconds{nl}Conversion duration 600 seconds	마법진 지속시간 #{CaptionTime}#초{nl}개종 지속시간 600초	SKILL_20150729_004963	Magic Circle Duration: #{CaptionTime}# seconds	마법진 지속시간 #{CaptionTime}#초	
+SKILL_20170912_009600	Magic Circle Duration #{CaptionTime}# seconds{nl}Conversion duration 600 seconds	마법진 지속시간 #{CaptionTime}#초{nl}개종 지속시간 600초	SKILL_20150729_004963	Magic Circle Duration: #{CaptionTime}# seconds	마법진 지속시간 #{CaptionTime}#초	
```

Comment:
Colon after `Magic Circle Duration`
`Conversion duration 600 seconds` -> `Conversion Duration: 600 seconds` to keep consistency.